Source/WebCore/Modules/highlight/Highlight.cpp

```diff
@@ -48,6 +48,11 @@ static void repaintRange(const AbstractRange& range)
     }
 }
 
+static bool repaintIfPriorityChanged(int oldPriority, int newPriority)
```

Comment:
Does this actually need to be a function? As far as I can tell its only used in one place

Reply:
I can put the logic in the place that is used instead of in a function. Is that preferred?

Reply:
It is, since the implementation is just an equality check